include/alpaka/exec/ExecCpuOmp4.hpp

```diff
@@ -188,7 +188,7 @@ namespace alpaka
                         {
                             Vec<dim::DimInt<1u>, TSize> const gridBlockIdx(b);
                             // When this is not repeated here:
-                            // error: �gridBlockExtent� referenced in target region does not have a mappable type
+                            // error: �gridBlockExtent� referenced in target region does not have a mappable type
```

Comment:
This file was utf8 encoded. I will fix the wrong symbols. A plain ASCII will is better, some compiler had problems with UTF8 (welcome to the 21th century :-)